Backend/main.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/go-redis/redis/v8"
+	"github.com/gorilla/mux"
+)
+
+var (
+	connection = connectToRedis()
+	ctx        = context.Background()
+)
+
+type Image struct {
+	URL string `json:"url"`
```

Comment:
If you expose variables in exposed struct you could provide a docs for it as well.